scaladoc/src/dotty/tools/scaladoc/SocialLinks.scala

```diff
@@ -19,5 +23,8 @@ object SocialLinks:
       case "gitter" => Left(errorPrefix + "For 'gitter' arg expected one argument: url")
       case "discord" if splitted.size == 2 => Right(Discord(splitted(1)))
       case "discord" => Left(errorPrefix + "For 'discord' arg expected one argument: url")
+      case LowercaseNamePattern() if splitted.size == 4 => Right(Custom(splitted(1), splitted(2), splitted(3)))
+      case LowercaseNamePattern() if splitted.size == 3 => Right(Custom(splitted(1), splitted(2), splitted(2)))
+      case LowercaseNamePattern() => Left(errorPrefix + "For 'custom' two minimum arguments are expected: url, white icon name, [dark icon name]")
```

Comment:
Nice, I think this is a lot nicer than what we had before with `custom`.

Reply:
Yes, thank you! Do you think I should write the pattern as custom() to specify that it's for custom icons or leave it like this to specify that it's a pattern for lowercase names?